examples/nffPktgen/generator/parseConfig.go

```diff
@@ -319,6 +336,16 @@ func parseData(in map[string]interface{}) (ret RawBytes, err error) {
 	return ret, fmt.Errorf("failed to parse data")
 }
 
+func (ipv4 *IPv4Config) UnmarshalJSON(data []byte) error {
```

Comment:
What about other configs lie IPv6Config for consistency?

Reply:
Ok